test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/PaginationSample.cs

```diff
@@ -5,51 +5,115 @@ namespace Tizen.NUI.Samples
 {
     public class PaginationSample : IExample
     {
+        private TextLabel board1, board2;
         private Pagination pagination1;
         private Pagination pagination2;
+        private View root;
+        private View parent1, parent2;
 
         private readonly int PAGE_COUNT = 5;
 
         public void Activate()
         {
             Window window = NUIApplication.GetDefaultWindow();
 
+            root = new View()
+            {
+                Size = new Size(1920, 1080),
+                BackgroundColor = new Color(0.7f, 0.9f, 0.8f, 1.0f),
+            };
+            window.Add(root);
+            window.KeyEvent += Window_KeyEvent;
+
+            parent1 = new View()
+            {
+                Position = new Position(500, 400),
```

Comment:
If you want to align items in center,
you can use LinearLayout and LinearAlignment or FlexLayout and ItemAlignment,Justification

Reply:
Romoved